sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -36,35 +36,32 @@
 class TrainingClient(object):
     def __init__(
         self,
-        config_file=None,
-        context=None,
-        client_configuration=None,
-        persist_config=True,
```

Comment:
@andreyvelich Is it possible to set a `persistent_config` with a `client_configuration`?

Reply:
I think `persistent_config` is used when we send client_configuration to `load_kube_config()` for Kubernetes python client.

Reply:
It makes sense.